src/views/datasets/buoy-telemetry/summary.vue

```diff
@@ -0,0 +1,327 @@
+<template>
+  <DashboardCard width="full">
+    <template #title>Current Conditions</template>
+    <template #subtitle>
+      <div class="content-container">
+        <div class="content">
+          This dashboard contains selected real-time climate data from buoys in
+          Narragansett Bay (NB). Measurements are collected using a suite of
+          sensors, logging data at 10-minute (Gill Maximet weather station,
+          PAR), 15-minute (Seabird Hydrocat and ECO), and 3-hour intervals
+          (Seabird Hydrocycle and SUNA). There may be up to a 12-hour lag in
+          available data as data stream into ERDDAP and through an automated QC
+          process. Explore and download more climate variables below, with data
+          available from the past three months. This work was funded by the
+          <a href="https://seagrant.gso.uri.edu/">Rhode Island Sea Grant</a> and
+          the
+          <a href="https://web.uri.edu/rinsfepscor/welcome/"
+            >Rhode Island Consortium for Coastal Ecology Assessment, Innovation,
+            and Modeling (RI C-AIM)</a
+          >.
```

Comment:
All of your external links should open links in a new tab. See examples in the `ma-buoy/sumary.vue` file.

```html
        <a
          href="http://www.dem.ri.gov/programs/emergencyresponse/bart/stations.php"
          target="_blank"
          rel="noreferrer noopener"
        >
          Narragansett Bay Fixed Station Monitoring Network (NBFSMN)</a
        >
```

In Signal, we added in a mini-component called `ExternalLink`. Could be worth doing here too! 

https://github.com/pph-collective/signal-app/blob/main/src/components/base/ExternalLink.vue

Reply:
great catch! I updated them and put an issue in for abstracting to a component